### ISSUE TITLE
laptop went kapoot

### ISSUE THREAD

**Issue by ihatemegabastard** [tonikelope/megabasterd#536]
after installing and starting download I went somewhere else when I returned my laptop went kapoor it was a nightmare I had to get it repaired and it costed me half of the laptop cost

**Comment by Akczht** [comment]
I'm sorry it seems to me like a your laptop issue, I could be wrong.

**Comment by ihatemegabastard** [comment]
I tried on window 9 laptop was 2 year old

**Comment by dasauto1** [comment]
I hope this post is 1 month and 17 days late. Your username makes me think otherwise.

**Comment by sfksuperman** [comment]
@ihatemegabastard Is Windows 9 even a thing? Bro are you using genuine windows or not?

@Akczht could you please confirm if this doesn't corrupt my windows as well? I'm using genuine windows 11 came with my laptop. Thanks.

**Comment by ihatemegabastard** [comment]
> @ihatemegabastard Is Windows 9 even a thing? Bro are you using genuine windows or not?
> 
> @Akczht could you please confirm if this doesn't corrupt my windows as well? I'm using genuine windows 11 came with my laptop. Thanks.

yeah you are right window 9 is not a thing I had 8 before this so there was not a need to check window version ever

**Comment by ihatemegabastard** [comment]
> I hope this post is 1 month and 17 days late. Your username makes me think otherwise.

I did not have a github account before this and yes this is not a hate post or anything like that I just wanted to tell be cautious and keep eye on task bar or something like that 
p.s- getting frustration out of head at the

**Comment by Grooves1210** [comment]
@ihatemegabastard 
What repair work did the laptop require?
